SU-HSDO/suhumsci
# Hsd8 193 custom date filter

# READY FOR REVIEW

# Summary
- Added extra options in date time filters to allow for an exception window. useful in course views.

# Needed By (Date)
- End of sprint: sept 13

# Urgency
- High

# Steps to Test

1. Checkout this branch
1. `composer update --prefer-source`
1. sync archaeology `blt drupal:sync --site=archaeology`
1. View the settings for the currently offered course view at `/admin/structure/views/view/hs_courses/edit/offered_list`
1. click on "settings" in near the filter for "Start Date"
1. ensure the exception values are configured.
1. ensure courses display in the view preview.

# See Also
- https://github.com/SU-SWS/stanford_courses/blob/7.x-4.x/modules/stanford_course_views/stanford_course_views.views_default.inc#L442
